ts/src/lbank2.ts

```diff
@@ -618,7 +636,7 @@ export default class lbank2 extends Exchange {
         params = this.omit (params, 'method');
         if (method === undefined) {
             const options = this.safeValue (this.options, 'fetchTrades', {});
-            method = this.safeString (options, 'method', 'publicGetTrades');
+            method = this.safeString (options, 'method', 'spotPublicGetTrades');
```

Comment:
@Dan-krm same here, why aren't we checking the type?

Reply:
@carlosmiei These methods that you're pointing out only have spot support just multiple different endpoints that can be used for spot that I carried over from the previous implementation. I haven't added the swap methods yet, in this PR I just added the swap endpoints and adjusted the naming of the spot endpoints and added some info to the docstrings.